integration-tests/integration/reportPages/enter-use-of-force-details.spec.js

```diff
@@ -73,7 +75,8 @@ context('Submitting use of force details page', () => {
         positiveCommunication: true,
         restraint: true,
         restraintPositions: ['STANDING'],
-        painInducingTechniques: true,
+        painInducingTechniquesUsed: true,
```

Comment:
Have changed to suffix with 'used' as I feel it is more relevant to a boolean type ~ it is a yes/no question rather than a set of values.

Reply:
As Andy pointed out, you can't change the name of the field because it won't line up with reports that have already been created.

Reply:
yes. you are correct . have redone reverted it